google/api/annotations.proto

```diff
@@ -36,6 +37,16 @@ extend google.protobuf.FileOptions {
   // Packaging information.
   // See `google/api/metadata.proto`.
   Metadata metadata = 1048;
+
+  // A representation of a resource.
+  // At a file level, this is generally used to define information for a
+  // resource from another API.
```

Comment:
It'll also be common to use it to define a information for a resource that doesn't have a proto message represeting it. E.g. ["Location"](https://github.com/googleapis/googleapis/blob/master/google/cloud/iot/v1/cloudiot_gapic.yaml#L42) resource in IOT.

Perhaps add to this comment:
...resource from another API; or for a resource that does not have an associated proto message.